### PR TITLE
Pkerpedjiev/df points docs

### PR DESCRIPTION
**Pull request by pkerpedjiev**
## Description

Added documentation for the [simplified labelled points tileset generator in higlass-python](https://github.com/higlass/higlass-python/pull/18)

Fixes #___

## Checklist

- [ ] Unit tests added or updated
- [x] Documentation added or updated
- [ ] Example added or updated
- [ ] Update schema.json if there are changes to the viewconf JSON structure format
- [ ] Screenshot for visual changes (e.g. new tracks)
- [ ] Updated CHANGELOG.md

### CODE REVIEW

File: docs/jupyter.rst [hunk 1]
```diff
@@ -281,40 +276,81 @@ Then we have to set up a data server to output the data in "tiles".
...
-        tileset_info=lambda: tsinfo,
-        tiles=lambda tile_ids: hgpo.format_data(
-                    hgut.bundled_tiles_wrapper_2d(tile_ids,
-                        lambda z,x,y,width=1,height=1: hgpo.tiles(df, 'x', 'y',
-                            tsinfo, z, x, y, width, height))))
+    ts = hgti.dfpoints(df, x_col='x', y_col='y')
 
-    # start the server
-    server = hfse.start([ts])
+    (display, server, viewconf) = higlass.display([
```
Comment (nvictus) [inline docs/jupyter.rst]: It is not Pythonic to use brackets in tuple function outputs.

Comment (flekschas) [inline docs/jupyter.rst]:
```suggestion
    display, server, viewconf = higlass.display([
```

Comment (flekschas) [reply docs/jupyter.rst]: FYI, there's a handy new feature for quickly fixing minor typos: the insert button (or cmd+g)

Comment (flekschas) [reply docs/jupyter.rst]: PS I fixed it :)

**Pull request merged**